pages/testnets/testnet_info.md

```diff
@@ -33,13 +36,13 @@ The above info can be found in this [`v4-testnets` repository](https://github.co
   },
   "stakeCurrency": {
     "coinDenom": "DV4TNT",
-    "coinDecimals": 6,
+    "coinDecimals": 18,
     "coinMinimalDenom": "dv4tnt",
```

Comment:
`coinMinimalDenom` should be `adv4tnt`

Reply:
lol I left a comment before I sent out the PR but never submitted my comment. I was gonna ask you this

updated!